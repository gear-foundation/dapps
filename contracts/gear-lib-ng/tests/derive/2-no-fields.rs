use gear_lib::StorageProvider;

struct Storage;

#[derive(StorageProvider)]
struct Contract {
    storage: Storage,
}

fn main() {}
