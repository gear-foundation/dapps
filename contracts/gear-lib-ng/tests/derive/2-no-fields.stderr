error: `StorageProvider` deriving does nothing if no field has the `#[storage_field]` attribute
 --> tests/derive/2-no-fields.rs:7:5
  |
7 |     storage: Storage,
  |     ^^^^^^^^^^^^^^^^^
